gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPicker.kt

```diff
@@ -135,6 +139,7 @@ internal fun AvatarPicker(
 internal fun AvatarPicker(uiState: AvatarPickerUiState, onEvent: (AvatarPickerEvent) -> Unit) {
     val context = LocalContext.current
     var loadingSectionHeight by remember { mutableStateOf(DEFAULT_PAGE_HEIGHT) }
+    var confirmAvatarDeletion by remember { mutableStateOf<Avatar?>(null) }
```

Comment:
```suggestion
    var confirmAvatarDeletion by rememberSaveable { mutableStateOf<Avatar?>(null) }
```

It would be nice to restore it when configuration changes, WDYT? 
THe `Avatar` can't be used here, could we store the `avatar_id` only?

Reply:
> It would be nice to restore it when configuration changes, WDYT?

I agree; good point. 

I've modified the viewmodel to work with the `imageId`. The dialog between configuration changes should be restored now.